File: app/src/main/java/ru/evteev/sportapp/domain/Sport.java
package ru.evteev.sportapp.domain;

public class Sport {
    public int id;
    public String name;

    public Sport(int id, String name) {
        this.id = id;
        this.name = name;
    }

    @Override
    public String toString() {
        return id + ". " + name;
    }
}

File: app/src/main/java/ru/evteev/sportapp/domain/Sport.kt
package ru.evteev.sportapp.domain

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey
import java.util.UUID

@Entity(tableName = "sports")
data class Sport(
    @PrimaryKey(autoGenerate = true )
    @ColumnInfo(name = "id")
    val id: Int = 0,

    @ColumnInfo(name = "name")
    var name: String = ""
)